src/backend/marsha/core/tests/test_api_organization.py

```diff
@@ -24,7 +24,11 @@ def test_create_organization_by_random_logged_in_user(self):
 
         jwt_token = AccessToken()
         jwt_token.payload["resource_id"] = str(user.id)
-        jwt_token.payload["user_id"] = str(user.id)
+        jwt_token.payload["user"] = {
```

Comment:
In this case you should probably have a get_token(user) method in test utils?

Reply:
What should return this method ? an `AccessToken` ?

Reply:
I think so yes? Saw it in Joanie and liked it. We are forging a looot of tokens in marsha's tests!